Conceptpower+Spring/src/test/java/edu/asu/conceptpower/core/impl/ConceptListManagerTest.java

```diff
@@ -0,0 +1,70 @@
+package edu.asu.conceptpower.core.impl;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import edu.asu.conceptpower.core.ConceptList;
+import edu.asu.conceptpower.db4o.IConceptDBManager;
+
+public class ConceptListManagerTest {
+
+    @Mock
+    private IConceptDBManager client = Mockito.mock(IConceptDBManager.class);;
+
+    @InjectMocks
+    private ConceptListManager conceptListManager;
+
+    @Before
+    public void init() {
+        MockitoAnnotations.initMocks(this);
+
+        List list = new ArrayList();
+        ConceptList conceptList = new ConceptList();
+        conceptList.setConceptListName("First List");
+        conceptList.setDescription("First List Description");
+        list.add(conceptList);
+
+        Mockito.when(client.getAllElementsOfType(ConceptList.class)).thenReturn(list);
+        Mockito.when(client.getConceptList("First List")).thenReturn(conceptList);
+        Mockito.when(client.getConceptList("Second List")).thenReturn(null);
+
+    }
+
+    @Test
+    public void getAllConceptListsTest() {

```

Comment:
You should write a second test to test the return value if there are no concept lists. If the current implementation returns an empty list, you need to check that.